app/lib/actions/katello/content_view/publish.rb

```diff
@@ -59,8 +59,13 @@ def plan(content_view, description = "", options = {importing: false, syncable:
 
             if options[:importing]
               handle_import(version, **options.slice(:path, :metadata))
-            elsif separated_repo_map[:pulp3_yum_multicopy].keys.flatten.present?
-              plan_action(Repository::MultiCloneToVersion, separated_repo_map[:pulp3_yum_multicopy], version)
+            else
+              if separated_repo_map[:pulp3_deb_multicopy].keys.flatten.present?
+                plan_action(Repository::MultiCloneToVersion, separated_repo_map[:pulp3_deb_multicopy], version)
+              end
+              if separated_repo_map[:pulp3_yum_multicopy].keys.flatten.present?
+                plan_action(Repository::MultiCloneToVersion, separated_repo_map[:pulp3_yum_multicopy], version)
+              end
```

Comment:
@ianballou That was the problem. It only did MultiCloneToVersion on either deb or yum, when it has to run for both.
Should be fixed now.

Thanks for testing that was a nice catch :+1: